### PR TITLE
Prepare built in private key interfaces.

### PR DESCRIPTION
**Pull request by Hathoriel**

### CODE REVIEW

File: src/model/request/TronMintTrc721.ts [hunk 1]
```diff
@@ -20,4 +21,23 @@ export class TronMintTrc721 extends MintErc721 {
   @Min(0)
   public feeLimit: number;
 
+  @IsNotEmpty()
```
Comment (samuelsramko) [inline src/model/request/TronMintTrc721.ts:24]: why is this here?

**Review by samuelsramko — CHANGES_REQUESTED** [review]

**Comment by Hathoriel** [comment]
Because i must keep tron same

Dne st 1. 9. 2021 13:04 uživatel ***@***.*** ***@***.***>
napsal:

File: src/model/request/TronMintTrc721.ts (hunk 1 above)
Comment (Hathoriel) [reply src/model/request/TronMintTrc721.ts:24]: Its needed to keep Tron same.

**Review by samuelsramko — APPROVED** [review]
